go.mod

```diff
@@ -22,8 +22,9 @@ require (
 	github.com/onsi/gomega v1.10.1
 	github.com/opencontainers/runc v1.0.0-rc2.0.20190611121236-6cc515888830 // indirect
 	github.com/openshift/api v3.9.1-0.20190927182313-d4a64ec2cbd8+incompatible
+	github.com/operator-framework/api v0.3.12
 	github.com/operator-framework/operator-lib v0.1.0
-	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200321030439-57b580e57e88
+	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200903182547-fddbf04ca175
```

Comment:
I think that can be removed if we avoid depending on `github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned/scheme` in `pkg/apis/addtoscheme_olm.go`.

Reply:
Yes, I took it out.  Thanks!

Reply:
Thanks.